docs-compose/overwatch_docs/content/DeployOverwatch/ConfigureOverwatch/Configuration.md

```diff
@@ -28,6 +28,10 @@ to you just leave it blank to avoid any strange issues when parsing the CSV.
 
 Please use the template for your version. If you are upgrading from 071x to 072x the upgrade script in the change log
 will upgrade the script for you.
+* 0.8.0.x+ ([.XLSX](/assets/DeployOverwatch/080x_overwatch_deployment_config_template.xlsx) |
+  [.CSV](/assets/DeployOverwatch/080x_overwatch_deployment_config_template.csv))
+  * Use this configuration for **multi account System Table Integration**
```

Comment:
Let's remove this line, and have all users in 08x running with the latest config, regardless of multi-account

Reply:
done